test/FieldMapper.test.js

```diff
@@ -0,0 +1,73 @@
+import { describe, expect, expectTypeOf, test } from 'vitest'
+import { FieldMapper } from '../lib/CapeMapper/FieldMapper.js'
+
+describe('FieldMapper', () => {
+  describe('constructor', () => {
+    test('When passed a simple configuration: an object is returned', () => {
+      const fieldMapper = new FieldMapper({ id: 'title', label: 'Title' })
+      expectTypeOf(fieldMapper).toBeObject()
+    })
+    /* eslint-disable no-unused-vars */
+    test('When not passed a label: an error is thrown', () => {
+      expect(() => {
+        const fieldMapper = new FieldMapper({ id: 'Title' })
+      }).toThrowError('Fields must have label, unless they are type \'ignore\'')
+    })
+
+    test('When not passed a label on a field of type "ignore": no error is thrown', () => {
+      expect(() => {
+        const fieldMapper = new FieldMapper({ id: 'Title', type: 'ignore' })
+      }).not.toThrowError()
+    })
+
+    test('When source_headings is a list including AUTO: throw an erorr', () => {
+      expect(() => {
+        const fieldMapper = new FieldMapper({ id: 'title', label: 'Title', source_headings: ['foo', 'AUTO'] })
+      }).not.toThrowError()
+    })
+    /* eslint-enable no-unused-vars */
+
+    test('When multiple is not set: default it to false', () => {

```

Comment:
Maybe rename the test here to be a statement of what happens (i.e. a specification) e.g.

_When multiple is not set then multiple defaults to false_